roles/acm_hypershift/README.md

```diff
@@ -27,45 +29,127 @@ api.<ah_cluster_name>.<ah_base_domain>
+| ah_download_cli         | True                                      | No        | Download some of the required binaries: oc, hcp.
+| ah_hcp_cli_path         |                                           | No        | Path to the hcp CLI, if `ah_download_cli` the role will get the binary from the Management Cluster
+| ah_ssh_key              | `sshKey` from Management cluster          | No        | SSH key to be added to the `authorized_keys` file in NodePool's instances
+| ah_disconnected         | False                                     | No        | Defines if the management cluster is in a disconnected (air-gapped) environment
+
+## Disconnected control planes
+
+For disconnected (air-gapped) environments it is required to already have the assets listed below available in a local registry instance. The [mirror_ocp_release](./mirror_ocp_release/README.md) role can help setup a disconnected registry with the needed assets.
+
+* RHCOs images
+* OCP release images
+
+### Variables for disconnected environments
+
+| Variable                | Default                                                    | Required  | Description
+| ----------------------- | ---------------------------------------------------------- | --------- | -----------
+| ah_ca_bundle_file       | `addittionalTrustBundle` applied to the Management cluster | No        | Path to the CA trust bundle applied to the Management cluster in disconnected environments
+| ah_hcp_annotations      | ""                                                         | No        | Annotations for the the HCP instance
+| ah_ics                  | IDMS from Management cluster                               | No        | File with additional Image Content Source Type (ICSP/IDMS) to be passed to the Hosted Cluster
+
+The role migrates the cluster's ICSP to IDMS, and those are passed to the Hosted Cluster if `ah_ics` is omitted and the the deployment is set as disconnected.
+
+Example of a file with Image Contents Sources definition.
+```yaml
+- mirrors:
+  - myregistry:4443/ocp-4.14/4.14.20
+  source: quay.io/openshift-release-dev/ocp-release
+- mirrors:
+  - myregistry:4443/ocp-4.14/4.14.20
+  source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+- mirrors:
+  - myregistry:4443/rhacm2/memcached-rhel8
+  source: registry.redhat.io/rhacm2/memcached-rhel8
+- mirrors:
+  - myregistry:4443/rhacm2/metrics-collector-rhel8
+  source: registry.redhat.io/rhacm2/metrics-collector-rhel8
+- mirrors:
+  - myregistry:4443/container-native-virtualization/kubevirt-tekton-tasks-disk-virt-customize-rhel9
+  source: registry.redhat.io/container-native-virtualization/kubevirt-tekton-tasks-disk-virt-customize-rhel9
+- mirrors:
+  - myregistry:4443/rhacm2/acm-prometheus-rhel8
+  source: registry.redhat.io/rhacm2/acm-prometheus-rhel8
+- mirrors:
+  - myregistry:4443/openshift-logging/fluentd-rhel9
+  source: registry.redhat.io/openshift-logging/fluentd-rhel9
+<snip>
```

Comment:
should this <snip> tag be, in fact, </snip>, and have <snip> defined at the beginning of the block code?